build.gradle

```diff
@@ -1,16 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = '1.4.21'
+    ext.kotlin_version = '1.5.31'
```

Comment:
Nice! Is this fully compatible with the coroutines' version we are using in this bootstrap?

Reply:
The kotlin version is no problem. A problem of duplicate dependencies arose when upgrading gradle. But I solved by excluding the module included by wolmo. 
This is the commit i am talking about: https://github.com/Wolox/WoloxAndroidBootstrap/pull/41/commits/b1ff2bcaaf5f49b882ff3ba6bf97acd6864d987d
All the tests run fine. So I assume there should be no problem.